insights/parsers/pci_rport_target_disk_paths.py

```diff
@@ -169,16 +174,13 @@ def parse_content(self, content):
                     temp_pci['host_channel_id_lun'] = line_sp[i - 1]
                 temp_pci['rport'] = rport
 
-                if len(temp_pci) == 5:
+                if len(temp_pci) == 6 and temp_pci not in pci:
```

Comment:
`temp_pci` is a `dict`, it's costly to compare it, the `in` operation will compare it with the items in the `pci` list one by one.

The better way to remove duplicated lines is to do the removal before re-construct it.

PS, I don't think there will be any duplicated lines, so it's enough for the Exception to only check the length of the `temp_pci`.